### DISCUSSION TITLE
Automatic switch between xDrip and Nightscout follower mode

### DISCUSSION THREAD

**Discussion by nightscout-ricsi** [NightscoutFoundation/xDrip#2230] [Ideas]
We are following our child's phone via xDrip sync, but lately we've had some disconnections. Some for hours, some for days.
It would be a great help to have the possibility to make xDrip automatically switch to NS following from xDrip sync, if it fails to sync for X minutes (where X is set by the user).

**Comment by Navid200** [comment]
Except if the cause of the problem is common like not having access to the internet.
There could be unintended consequences also.
Imagine if a parent has two diabetic kids and accidentally the automatic switch doesn't switch between the two accounts for the same kid but rather switches from one kid to the other.  There is no association between xDrip sync and Nightscount accounts.

I don't think this is such a good idea.  That's only my opinion.

**Reply by nightscout-ricsi** [reply]
We had this problem with proper internet access, with two followers failing.
There should be a hard link, what to switch. IF phoneJohn (spec security sync key) fails, THEN connect to NightscoutJohn (spec URL), and for another phone another NS.

**Comment by Navid200** [comment]
Can you explain a situation where Nightscout would fail, for example, once a day but xDrip sync would continue to work?

If Nightscout fails for some reason, the follower can switch to xDrip sync and vice versa.
But, this is not something that happens frequently.
Usually, people use one or the other.  

Can you explain why one of those two would fail and come back frequently?

**Comment by mtumashenko** [comment]
Found this topic while looking for this option.  I can't find a way to share insulin/carbs via nightscout cloud, only xdrip in follower mode shows this data. 
Maybe I'm doing something wrong with nightscout upload configuration, but auto switch to xdrip follower would nice.
Update: found that nightscout have this data (see it on web), so it's my xdrip on a follower phone problem. Though don't know where the problem is.

**Reply by Navid200** [reply]
Your question has nothing to do with the topic of this thread.  Of course xDrip is common and Nightscout is common.
But, it helps if a thread is about the same subject.  That was the whole point of asking people to use these forums instead of using the Gitter room.

Please open a new thread and I will search and if I find an associated issue, will point you to it.
